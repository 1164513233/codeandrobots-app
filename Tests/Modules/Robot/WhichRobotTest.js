import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import WhichRobotScreen from 'App/Modules/Robot/WhichRobot/Screen'

describe('WhichRobot', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(
      <WhichRobotScreen
        navigation={{navigate: () => {}}}
        showNotReadyModal={false}
        onHideNotReadyModal={() => {}}
        onPress={() => {}} />)
  })

  it('Component renders correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
