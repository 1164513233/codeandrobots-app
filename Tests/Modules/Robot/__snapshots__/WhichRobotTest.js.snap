// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhichRobot Component renders correctly 1`] = `
<Container>
  <List
    linedRows={true}
    scrollable={true}
  >
    <CardListItem
      image={
        Object {
          "process": [Function],
        }
      }
      text="The open source robot that you can build yourself"
      title="Otto DIY"
    />
    <CardListItem
      image={
        Object {
          "process": [Function],
        }
      }
      text="The world's cutest open source robotic kitten"
      title="Nybble"
    />
    <CardListItem
      image={
        Object {
          "process": [Function],
        }
      }
      text="No robot, no problem, simulator to the rescue"
      title="Robot Simulator"
    />
    <CardListItem
      image={
        Object {
          "process": [Function],
        }
      }
      text="Letâ€™s see if your robot has what it takes"
      title="Custom Robot"
    />
  </List>
</Container>
`;
